intro_to_R_tidyverse/00-rstudio_guide.md

```diff
@@ -0,0 +1,80 @@
+# A quick guide to RStudio
+
+Our course material uses [R Markdown Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
+Opening and running R Markdown (`.Rmd`) files requires [RStudio](https://www.rstudio.com/products/RStudio/).
+RStudio is an [integrated development environment](https://en.wikipedia.org/wiki/Integrated_development_environment) or IDE that makes it easier to program in R. 
+In our case, we use [Docker](https://www.docker.com/) containers with RStudio on them (specifically from the [Rocker Project](https://www.rocker-project.org/)) to ensure that all course participants have the same environment.
+A detailed discussion of Docker is outside the scope of this guide.
+**Here, our objective is to provide a brief guide to getting started with RStudio.**
+We assume you have either installed R and RStudio locally or are using an appropriate Docker container.
+
+Note there is an excellent _Take a tour of RStudio’s IDE_ video from RStudio [here](https://www.rstudio.com/products/rstudio/).
+
+## Anatomy of RStudio
+
+_These screenshots were generated using a Docker container._
+
+1. When you open an RStudio session for the first time, it will look like this.
+
+![overview](screenshots/00-rstudio.png)
+
+2. The **Console** is where you will type commands that will be carried out by `R`.
+
+![console](screenshots/01-console.png)
+
+3. You can switch to a **Terminal** where commands will be carried out by clicking the `Terminal` tab to the right of the `Console` tab. 
+This is a [command-line interface](https://en.wikipedia.org/wiki/Command-line_interface).
+Commands here will not be carried out by R.
+
+![terminal](screenshots/02-terminal.png)
+
+4. The **Environment** pane displays all of the objects and functions in our global environment.
+Here we can see that we have a `data.frame` called `iris` in our global environment.
+
+![environment](screenshots/03-environment.png)
+
+5. You can click on `iris` to see what the `data.frame` looks like.
```

Comment:
Technically you don't have to click *specifically* on the grid icon, you can click anywhere on the name of the object and it does the `View()` command.

Reply:
Updated the screenshot